source/deprecations/v1.x.html.md

```diff
@@ -1172,6 +1172,62 @@ Object.freeze(c);
 c.swapNames();  // EXCEPTION
 ```
 
+### `Copyable.frozenCopy`
+
+Just as the `Freezable` mixin is deprecated in favor of functionality in
+core JavaScript, the `frozenCopy` method of the Copyable mixin is also
+deprecated in favor of `Object.freeze()`.
+
+Replace the following code:
+
+```javascript
+const Obj = Ember.Object.extend(Freezable, Copyable, {
+  copy() {
+    return Obj.create();
+  }
+});
+
+const copy = Obj.create().frozenCopy();
+```
+
+with:
+
+```javascript
+const copy = Object.freeze();
```

Comment:
I expect you want:

```js
'use strict';

let a = {};
Object.isFrozen(a); // => false
Object.freeze(a);
Object.isFrozen(a); // => true
a.foo = 'baz'; // => throws TypeError
```

It may also be worth pointing to the documentation for [Object.seal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/seal) and [Object.freeze](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze)